package.json

```diff
@@ -34,9 +40,12 @@
     "react-native-modal": "^4.1.1",
     "react-native-nfc-manager": "0.0.6",
     "react-native-progress": "^3.4.0",
+    "react-native-randombytes": "^3.0.0",
     "react-native-swiper": "^1.5.13",
     "react-native-vector-icons": "^4.4.2",
-    "react-navigation": "^1.0.0-beta.25"
+    "react-navigation": "^1.0.0-beta.25",
+    "readable-stream": "^2.3.3",
+    "vm-browserify": "^0.0.4"
```

Comment:
Do we actually need this as a dependency? Not a dev dependency?

Reply:
its a dependency of asn1.js which used by node-rsa